internals/_init.php

```
@@ -0,0 +1,271 @@
+        return true;
+    } else {
+        return false;
+    }
+}
+
+function check_path_query(): string {
+    // ToDo: Also canonicalize
+    
+    if (!isset($_GET['path'])) {
+        http_response_code(400);
+        die("`path` parameter is missing");
```

PHPMD (2.15.0)

[error] The function check_path_query() contains an exit expression. (undefined)

(ExitExpression)